app/validators/base_sub_model_validator.rb

```diff
@@ -35,6 +35,7 @@ def validate_has_one_associations(record)
   def validate_collection_for(record, association_name)
     collection = record.__send__(association_name)
     collection.each_with_index do |associated_record, i|
+      next if associated_record.marked_for_destruction?
```

Comment:
⚠️  This is one of the new **code climate** _offended_ files due to this change?! 😕